templates/search/results-did-you-mean-en.html

```diff
@@ -0,0 +1,34 @@
+---
+{
+	"title": "Search results (did you mean)",
+	"language": "en",
+	"altLangPage": "results-did-you-mean-fr.html",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"dateModified": "2025-07-25",
+	"share": "false",
+	"deptfeature": false,
+	"nositesearch": true
+}
+---
+<form class="mrgn-tp-sm" role="search">
+	<div class="input-group mrgn-tp-lg">
+		<label class="wb-inv" for="sch-inp-ac">Search Government of Canada websites</label>
+		<input id="sch-inp-ac" class="form-control" value="" name="q" autocomplete="off" spellcheck="false" type="search" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+		<button class="btn btn-primary btn-small" type="submit"> <span class="glyphicon-search glyphicon" aria-hidden="true"></span> <span class="wb-inv">Search</span> </button>
+		</span>
+	</div>
+	<p id="gc-pi" class="mrgn-tp-md">Don’t include personal information (telephone, email, SIN, financial, medical, or work details).</p>
+</form>
+
+<section id="wb-land">
+	<h2>["Query summary" or "No results" heading structure]</h2>
+	<div id="did-you-mean">
+		<p class="h5">Did you mean <button class="btn btn-lg btn-link p-0 mrgn-bttm-sm" type="button">%[Corrected Query]</button> ?</p>
```

Comment:
Please remove the % sign
```suggestion
		<p class="h5">Did you mean <button class="btn btn-lg btn-link p-0 mrgn-bttm-sm" type="button">[Corrected Query]</button> ?</p>
```